Makefile

```diff
@@ -0,0 +1,2 @@
+test:
+	cd /var/lib/GA_mock/SilMock/tests; /var/lib/GA_mock/SilMock/tests/phpunit
```

Comment:
These would commonly be on two separate lines:
```suggestion
	cd /var/lib/GA_mock/SilMock/tests
	/var/lib/GA_mock/SilMock/tests/phpunit
```

Reply:
> These would commonly be on two separate lines:

wouldn't be in the correct directory if separate lines.